src/solve/test.rs

```diff
@@ -1399,3 +1399,159 @@ fn inapplicable_assumption_does_not_shadow() {
         }
     }
 }
+
+#[test]
+fn auto_trait_without_impls() {
+    test! {
+        program {
+            #[auto] trait Send { }
+
+            struct i32 { }
+            struct Vec<T> { }
+        }
+
+        goal {
+            i32: Send
+        } yields {
+            "Unique"
+        }
+
+        goal {
+            forall<T> {
+                Vec<T>: Send
```

Comment:
So this through me for a bit, but I guess it's true because `struct Vec<T> { }` has no fields